src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -168,6 +168,9 @@ public override void WriteWarningLine(string message)
 
     internal class CommandLineParameterParser
     {
+        private const int _maxPipePathLengthLinux = 108;
```

Comment:
I believe the convention for const is to use Pascal Casing.  This shows up as a style cop issue (I know the code base doesn't follow this consistently...)

```suggestion
        private const int MaxPipePathLengthLinux = 108;
```

Reply:
Fixed.